keps/sig-auth/1205-bound-service-account-tokens/README.md

```diff
@@ -265,53 +258,176 @@ service account token on behalf of pods running on that node. The
 NodeRestriction admission controller will require that these tokens are pod
 bound.
 
-### Safe Adoption
+### ServiceAccount Admission Controller Migration
+
+#### Prerequisites
+
+Before migration to a version with `BoundServiceAccountVolume=true`, cluster
+operators should make sure:
+
+1.  Set feature gate `TokenRequest=true`. (default to `true` since 1.12)
+
+    - This feature requires the following flags to the API server:
+      - `--service-account-issuer`
+      - `--service-account-signing-key-file`
+      - `--service-account-key-file`
+      - `--api-audiences` (default to `--service-account-issuer`)
+
+2.  Set feature gate `TokenRequestProjection=true`. (default to `true` since
+    1.12)
+
+3.  Update all workloads to newer version of officially supported Kubernetes
+    client libraries to reload token:
+
+    - Go: >= v11.0.0
+    - Python: >= v12.0.0
+    - Java: >= v9.0.0
+    - Javascript: >= v0.10.3
+    - Ruby: master branch
+    - Haskell: master branch
+
+    For community-maintained client libraries, feel free to contribute to them
+    if the reloading logic is missing.
+
+    **Note**: If having trouble in finding places using in-cluster config
+    completely, cluster operators can specify flag
+    `--service-account-extend-token-expiration` to kube apiserver to allow
+    tokens have longer expiration temporarily during the migration. Any usage of
+    legacy token will be recorded in both metrics and audit logs. After fixing
+    all the potentially broken workloads, don't forget to remove the flag so
+    that the original expiration settings are honored.
+
+    - Metrics: subsystem: serviceaccount, name: stale_tokens_total
```

Comment:
nit: switch this to the full constructed prometheus metric name

Reply:
is it "serviceaccount_stale_tokens_total"? done.l